containers/ddev-webserver/Dockerfile

```diff
@@ -99,11 +99,11 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -qq install -o Dpkg::Options::="--for
 ADD ddev-webserver-dev-base-files /
 RUN source /tmp/ddev/vars && curl -sSL https://github.com/drud/MailHog/releases/download/v${MAILHOG_VERSION}/MailHog_${MAILHOG_ARCH} -o /usr/local/bin/mailhog; 
 
-RUN curl -sSL https://github.com/pantheon-systems/terminus/releases/download/$(curl --silent "https://api.github.com/repos/pantheon-systems/terminus/releases/latest" | perl -nle'print $& while m{"tag_name": "\K.*?(?=")}g')/terminus.phar --output /usr/local/bin/terminus && chmod 777 /usr/local/bin/terminus
-RUN curl -sSL https://github.com/platformsh/platformsh-cli/releases/download/$(curl --silent "https://api.github.com/repos/platformsh/platformsh-cli/releases/latest" | perl -nle'print $& while m{"tag_name": "\K.*?(?=")}g')/platform.phar --output /usr/local/bin/platform && chmod 777 /usr/local/bin/platform
-RUN curl -sSL --output /usr/local/bin/acli https://github.com/acquia/cli/releases/latest/download/acli.phar && chmod 777 /usr/local/bin/acli
-RUN curl -sSL https://github.com/backdrop-contrib/drush/releases/download/${BACKDROP_DRUSH_VERSION}/backdrop-drush-extension.zip -o /tmp/backdrop-drush-extension.zip && unzip -o /tmp/backdrop-drush-extension.zip -d /var/tmp/backdrop_drush_commands
 RUN curl -sSL --output /usr/local/bin/phive "https://phar.io/releases/phive.phar" && chmod 777 /usr/local/bin/phive
+RUN phive install --force-accept-unsigned --target /usr/local/bin/ --temporary pantheon-systems/terminus
```

Comment:
Nice. Now if we could get these signed properly we'd really be somewhere. https://github.com/drud/ddev/issues/2969 is a super-important initiative to improve defenses against supply-chain exploits. Not sure what's required.

I note that there's also a `phive install --global` which might serve the same as your --target /usr/local/bin

Reply:
None of these PHARs are signed :( But the global flag is a good idea

Reply:
> I don't understand why magerun, magerun2, terminus, platform, and acli can't be installed with phive. Remind me?

Coming back to magerun and magerun2: they can also be installed using PHIVE, but they would be saved under their "original" names `n98-magerun` and `n98-magerun2`. Adding a symlink to these would be possible, I think